ParisBNF/abb/BNFabb202.xml

```diff
@@ -0,0 +1,483 @@
+                                  <seg type="ink">Red.</seg>
+                                  <seg type="rubrication">Rubricated chapter numbers (filled only on some folia, blank spaces left unfilled in the most cases),
+                                    some incipits.</seg>
+                             <desc>Crude, by an unskilled scribe.</desc>
+                             </handNote>
+
+                             <handNote xml:id="h3" script="Ethiopic">
+                               <locus from="56ra" to="68vb"/>
+                               <locus from="72ra" to="73rb"/>
+                            <seg type="script">Mid-18th - mid-19-century raqiq script.</seg>
+                                  <seg type="ink">Black.</seg>
+                                  <desc>Careful, somewhat irregular, by a trained scribe. Right sloping. Tops are horizontal.
+                            Slender, rounded letter shapes. 13-14 characters per line (<locus target="#65r"/>)</desc>
+                           </handNote>
+
+                             <handNote xml:id="h4" script="Ethiopic">
+                               <locus from="69rb" to="72rb"/>
+                               <locus from="73vb" to="81vb"/>
+                               <seg type="script">Mid-18th - mid-19th-century raqiq script.</seg>
+                                    <seg type="ink">Black.</seg>
+                                    <desc>Not very regular, by a trained scribe.
+                               Broadly spaced, upright to slightly left sloping,
+                               bulky script. Tops are slightly inclined to the links.
+                               Broad letter shapes, tendency to rectangular forms. 12-13 characters per line
+                             (<locus target="#69rb"/>). In many letters the elements are disjoined.
+                           Word dividers are very tiny.</desc>
+                                </handNote>
+                        </handDesc>
+
+                        <decoDesc>
+                          <decoNote type="ornamentation" xml:id="d1">
+                          <locus target="#1r"/>
+                          <desc>Monochrome (red) ornamental band.</desc>
+                        </decoNote>
+                        </decoDesc>
+
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1ra"/>
+                               <desc type="Unclear">A short prayer defying Satan.</desc>
+                                 <q xml:lang="gez">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ። በቅደስት፡ ሥላሴ፡
+                                 እንዘ፡ ኣምን፡ ወእት፡ መኃፀን፡ እከህደከ፡ ሰይጣን፡ በቅድመ፡ ዛቲ፡ እምየ፡ ቅድስት፡ ቤተ፡ ክርስቲያን፡ እንተ፡
+                               ይእቲ፡ ስማዕየ፡ ማርያም። ጽዮን፡ ለዓለመ፡ ዓለም።</q>
+                                   <note>Written in a good hand.</note>
```

Comment:
Ok, I see that you like `<note>` in additions :) In my understanding, it is really not necessary, because the information given in `<note>` can be give in `<desc>` . But maybe my understading is limited. Is there a specific reason why you prefer to have `<note>` ?

Reply:
In "note" I used to place the information that pertain on physical aspect of the note including hands and text arrangement; the information above in <desc> in for me an equivalent of the "title" and explains the content. Not good? :)

Reply:
Aha! This is a bit idiosyncratic, as it was never discussed, I think. For me, the `@type` of `<desc>` or its first words would serve as the title, while the description would follow continuously. But if you want a more precise structure, we can find a solution for this (I know you are not the only one who wants to use note in additions). For me, there is nothing intuitive for stating that note serves for the physical description. Maybe @PietroLiuzzo has an idea? Do you think that such a structure could be introduced (with note or otherwise)?

Reply:
you can do both, I do not think we are introducing anything. the use of note is an additional layer, you can imagine you are adding a footnote, while without it you are continuing the description. I personally see no need to dictate one way or another, of course a consistent use would be better, but we do have already quite a lot of additional rules and this does not seem to me terribly decisive in any context of use for the application.

Reply:
I do not know. Should it be fixed at all? I now remember that I I use <description> here because it cannot be assigned to any of the fixed additional note typologies, so its quasi-title + content needs some exaplanation. For the rest, I really use "note".

Reply:
It is not fixed now. For me, it would seem easier to read everything in desc (I also think that you don't intend the note to have a footnote like character), than to introduce more elements, but do it as you like.